dev/packages/example/base/0.2.0/elasticsearch/component-template/logs-settings.json

```diff
@@ -5,15 +5,16 @@
                 "lifecycle": {
                     "name": "logs-default"
                 },
+                // Best compression optimises for storage
```

Comment:
@jpountz This works with ES but most validators and Github are not very happy with this. What does ES use to validate these?

Reply:
Do you mean validating JSON files that include comments? We're relying on the library we use for JSON parsing, Jackson. https://github.com/FasterXML/jackson

Reply:
Yes, exactly. Most JSON validators I have seen so complain if they see comments. But potentially my workaround is to just use yaml instead (see other change in this PR).

Reply:
I'll bring this question to our Fix-it meeting to discuss what should be the way forward.

Reply:
@jpountz Not sure if ES needs to fix anything here as in ES things work just as expected (as long as I don't try to validate it somewhere :-) ).

Reply:
Maybe not indeed, but I think it's important to at least raise awareness about this problem, ie. the fact that we allow comments in JSON doesn't mean that it's easy for consumers of the Elasticsearch APIs to leverage this feature.